### DISCUSSION TITLE
Welcome to pulse Discussions!

### DISCUSSION THREAD

**Discussion by finsberg** [finsberg/pulse#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramayzoraiz** [comment]
Dear Finsberg,
                     Excellent work. I was wondering why I_8fs in strain_energy function(in HolzapfelOdgen.py) is not isochoric in terms of F (deformation tensor)?

**Reply by finsberg** [reply]
This looks like a bug. Would you like to submit a pull-request to fix it? :)

**Reply by ramayzoraiz** [reply]
Sure
